vllm/config.py

```diff
@@ -33,14 +33,21 @@
 _EMBEDDING_MODEL_MAX_NUM_BATCHED_TOKENS = 32768
 _MULTIMODAL_MODEL_MAX_NUM_BATCHED_TOKENS = 5120
 
+Task = Literal["generate", "embed"]
```

Comment:
What's your thought about using enums instead of strings?

Reply:
I would use strings unless there is a need to attach additional data for each task. Otherwise, we will have to spend additional effort converting between string and enum at the API level.